pkg/util/kubernetes/apiserver/leaderelection/leaderelection.go

```diff
@@ -89,13 +91,19 @@ func ResetGlobalLeaderEngine() {
 
 // GetLeaderEngine returns a leader engine client with default parameters.
 func GetLeaderEngine() (*LeaderEngine, error) {
-	return GetCustomLeaderEngine("", defaultLeaderLeaseDuration)
+	return GetCustomLeaderEngine(context.Background(), "", defaultLeaderLeaseDuration)
```

Comment:
defaultLeaderLeaseDuration is still 60s? Can we reduce it to 1s?

Reply:
1s is the same as releasing the lease so it might break leader election (every pod will try to acquire the lease every second and the leader will change every second)